### ISSUE TITLE
Export profile can error

### ISSUE THREAD

**Issue by ccchapman** [solspace/craft-freeform#322]
**Describe the bug or issue you're experiencing**

An export profile will sometimes fail.

**Steps to reproduce**

1. Create a new form with a submit button
2. Make a submission to the form
3. Add a table field to the form
4. Run the export profile

**Expected behavior**

Export is successful

**Actual behavior**

An error occurs, until you manually save the submission from step 2.

```
TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /srv/dev.project.com/vendor/solspace/craft-freeform/packages/plugin/src/Library/Export/Objects/Row.php:33
Stack trace:
#0 /srv/dev.project.com/vendor/solspace/craft-freeform/packages/plugin/src/Library/Export/AbstractExport.php(182): Solspace\Freeform\Library\Export\Objects\Row->addColumn(Object(Solspace\Freeform\Library\Export\Objects\Column))
#1 /srv/dev.project.com/vendor/solspace/craft-freeform/packages/plugin/src/Library/Export/AbstractExport.php(36): Solspace\Freeform\Library\Export\AbstractExport->parseSubmissionDataIntoRows(Array)
#2 /srv/dev.project.com/vendor/solspace/craft-freeform/packages/plugin/src/Services/Pro/ExportProfilesService.php(239): Solspace\Freeform\Library\Export\AbstractExport->__construct(Object(Solspace\Freeform\Form\Types\Regular), Array, false, false)
#3 /srv/dev.project.com/vendor/solspace/craft-freeform/packages/plugin/src/Controllers/Pro/ExportProfilesController.php(159): Sols
```

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.7.38
- Freeform Version: 3.13.7
- Freeform Edition: Pro

**Additional context**

**Comment by kjmartens** [comment]
Sorry for the trouble @ccchapman. We'll work on a fix for this soon. 🙂

**Comment by kjmartens** [comment]
Hi @ccchapman, this should be resolved in Freeform 3.13.8 now. 🙂
